CHANGELOG_PENDING.md

```diff
@@ -1,5 +1,7 @@
 ### Improvements
 
+- [auto/dotnet] - Add support for `--exact` and `--server` with `pulumi plugin install` via Automation API.
```

Comment:
Mind adding a note about the signature change for `InstallPluginAsync` (binary breaking, but largely source compatible)?

Reply:
Done!